eng/pipelines/spec-gen-sdk.yml

```diff
@@ -32,9 +32,9 @@ parameters:
     
 trigger: none
 pr:
-  paths: 
+  branches:
```

Comment:
Do we need to list the branches at all if it is just '*'?

Reply:
This is the latest [test PR](https://github.com/raych1/azure-rest-api-specs/pull/61)